### PR TITLE
8-create login page

### PR DESCRIPTION
**Pull request by rzgholizadeh**
# Pull Request Template

## **Checklist**

- [ ] **Code Quality**: Code has been reviewed and meets project standards.
- [ ] **Tests**: Added or updated tests to cover changes.
- [ ] **Documentation**: Updated relevant documentation (e.g., README, inline comments, or external docs).
- [ ] **CI/CD**: Ensured all CI checks pass (linting, tests, etc.).
- [ ] **Dependencies**: Checked for new dependencies and updated `package.json` and `package-lock.json` if necessary.

### CODE REVIEW

**Comment by aminations** [comment]
Can you please add instructions on how to run the tests for this PR?

File: src/components/Home.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ import React from "react";
 import ContructionNoticeCreate from "./ContructionNoticeCreate";
 
 const Home = () => {
-    return <div><ContructionNoticeCreate /></div>
+    return <div>
+        <ContructionNoticeCreate />
+    </div>
```
Comment (aminations) [inline src/components/Home.js:7]: Can we also fix these please? Maybe you can merge main first as I've already merged the PR for this type to the main.

File: src/components/Register.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Register = () => {
+    return <div>
+        Hehe we cannot register you at the moment :P
+    </div>
```
Comment (aminations) [inline src/components/Register.js:6]: love it :)

**Review by aminations — CHANGES_REQUESTED** [review]
Sounds good to me. Let's do the following?

- Merge main to this branch to have the latest updates on the main here.
- Let's also add extra empty line to the end of each file.
- Can you add instructions about how to test this.

File: src/components/Register.js (hunk 2 above)
Comment (rzgholizadeh) [reply src/components/Register.js:6]: Always say the truth.

File: src/components/Home.js (hunk 1 above)
Comment (rzgholizadeh) [reply src/components/Home.js]: Fixed the conflict. Should be good for main.

**Comment by rzgholizadeh** [comment]
@aminations 
What do you mean by "add instruction about how to test"? As in how to run unit tests?

**Comment by aminations** [comment]
Yes, how can I run the unit tests for this?

**Review by aminations — APPROVED** [review]

**Comment by rzgholizadeh** [comment]
> Yes, how can I run the unit tests for this?

I'll add it in the readme.
But it is defined in the package.js file. You run 'npm test'.

**Comment by aminations** [comment]
@rzgholizadeh, I tried `npm test` but it doesn't work!
Screenshot 2024-12-31 at 12 32 32 PM

**Pull request merged**